pom.xml

```diff
@@ -207,6 +207,10 @@ THE SOFTWARE.
           <groupId>org.apache.maven</groupId>
           <artifactId>maven-core</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.eclipse.sisu</groupId>
+          <artifactId>org.eclipse.sisu.plexus</artifactId>
+        </exclusion>
```

Comment:
This sort of POM hacking is a good sign that you should be writing a `RealJenkinsRule` test to prove that the basic functionality of the plugin still works.